pkgs/development/python-modules/opcua-widgets/default.nix

```diff
@@ -0,0 +1,34 @@
+{ pkgs
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, python3Packages
+}:
+
+buildPythonPackage rec {
+  version = "0.6.1";
+  pname = "opcua-widgets";
+
+  src = fetchFromGitHub {
+    owner = "FreeOpcUa";
+    repo = pname;
+    rev = version;
+    hash = "sha256-ABJlKYN5H/1k8ynvSTSoJBX12vTTyavuNUAmTJ84mn0=";
+  };
+
+  disabled = pythonOlder "3.10";
+
+  propagatedBuildInputs = [
+    python3Packages.pyqt5
+    python3Packages.asyncua
+  ];
+
+  doCheck = false;
```

Comment:
Why? We want to run tests

Reply:
Then please add a comment that the tests are broken

Reply:
Okay added it